test/debate.js

```diff
@@ -90,6 +99,236 @@ describe('Debate test', () => {
         });
     });
 
+    describe("Debate client functions", () => {
+        let client;
+        let id;
+        let debate;
+
+        before((done) => {
+            admin = io.connect(`http://localhost:${SocketConfig.SOCKET_PORT}${SocketConfig.ADMIN_NAMESPACE}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+                query: {
+                    password: `${SocketConfig.ADMIN_PASSWORD}`
+                }
+            });
+
+            let debateInfo = {
+                title: 'My new debate',
+                description: 'Test debate'
+            };
+            admin.emit("newDebate", debateInfo, (debateID) => {
+                id = debateID;
+                done();
+            });
+        });
+
+        beforeEach((done) => {
+            debate = debateManager.nspAdmin.getActiveDebate(id);
+
+            client = io.connect(`http://localhost:${SocketConfig.SOCKET_PORT}${SocketConfig.DEBATE_NAMESPACE_PREFIX}${id}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+            });
+
+            client.on('connect', () => {
+                done();
+            })
```

Comment:
End of instruction char ';' is missing.
```suggestion
            });
```